bonus-opdracht.js

```diff
@@ -0,0 +1,29 @@
+const sortTvButton = document.getElementById("sort-tv");
+sortTvButton.addEventListener("click", showSortedTv);
+
+function showSortedTv(){
+    sortPrice(inventory);
+    generateTvList(inventory);
+}
+
+
+const ambiLightButton = document.getElementById("ambi-light");
+ambiLightButton.addEventListener("click", showAmbiLight);
+
+function showAmbiLight(){
+    const tvAmbient = inventory.filter((hasAmbient ) =>{
+        return hasAmbient.options.ambiLight === true;
```

Comment:
Omdat `hasAmbient.options.ambiLight` op zichzelf al een boolean is hoeft deze niet gecheckt te worden op `true`
```suggestion
        return hasAmbient.options.ambiLight;
```